### PR TITLE
Incorporate design feedback into population chart

### PR DESCRIPTION
**Pull request by Catacola**
## Description of the change

- Move projected label to lower right
- Move projected shading to behind lines
- Add test for tooltip component
- Rename all instances of `Timeseries` to `TimeSeries`
- Use aliases for all colors

## Type of change

- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

Part of #986

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

**Comment by daschi** [comment]
Does this also include the design polish from this review?

**Comment by Catacola** [comment]
It's the plot work for that. We still need to address the filter and summary card feedback.

File: src/core/PopulationTimeSeriesChart/index.tsx [hunk 1]
```diff
@@ -19,16 +19,17 @@ import React from "react";
 import { useLocation } from "react-router-dom";
 import { scaleTime } from "d3-scale";
 import { observer } from "mobx-react-lite";
+import { ResponsiveXYFrame } from "semiotic";
```
Comment (daschi) [inline src/core/PopulationTimeSeriesChart/index.tsx:22]: What led you to switch back to this?

File: src/core/PopulationTimeSeriesChart/PopulationTimeSeriesLegend.scss [hunk 2]
```diff
@@ -1,3 +1,5 @@
+@import "../CoreConstants";
+
 .PopulationTimeseriesLegend {
```
Comment (daschi) [inline src/core/PopulationTimeSeriesChart/PopulationTimeSeriesLegend.scss:3]: Should we also update the classname?

File: src/core/PopulationTimeSeriesChart/__tests__/PopulationTimeSeriesLegend.test.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
+// Recidiviz - a data platform for criminal justice reform
```
Comment (daschi) [inline src/core/PopulationTimeSeriesChart/__tests__/PopulationTimeSeriesLegend.test.js:1]: 🥳

**Review by daschi — APPROVED** [review]
Looks great!!

File: src/core/CoreConstants.scss [hunk 4]
```diff
@@ -41,7 +41,7 @@ $white: #fff;
 $box-shadow-color-30: rgba(19, 44, 82, 0.3);
 $box-shadow-color-20: rgba(19, 44, 82, 0.2);
 $inset-shadow-30: inset 0px -1px 1px $box-shadow-color-30;
-$pine-1: rgba(1, 35, 34, 1);
+$pine-1: rgb(1, 35, 34);
```
[the remark this answers is not in the document]
Comment (Catacola) [reply src/core/CoreConstants.scss:44]: Could have been my merge as well. I told intellij to do its magic and forgot to double check.

File: src/core/PopulationTimeSeriesChart/PopulationTimeSeriesLegend.scss [hunk 5]
```diff
@@ -16,18 +18,16 @@
   }
 
   &__Line {
+    stroke: #25636F;
```
[the remark this answers is not in the document]
Comment (Catacola) [reply src/core/PopulationTimeSeriesChart/PopulationTimeSeriesLegend.scss:21]: yup!

File: src/core/PopulationTimeSeriesChart/index.tsx (hunk 1 above)
Comment (Catacola) [reply src/core/PopulationTimeSeriesChart/index.tsx:22]: I decided it was worth the safety and hints and autocompletions that I did get for the unsuppressed fields and that adding 5 or so suppressions across the entire class was a fair tradeoff.

File: src/core/PopulationTimeSeriesChart/PopulationTimeSeriesLegend.scss (hunk 2 above)
Comment (Catacola) [reply src/core/PopulationTimeSeriesChart/PopulationTimeSeriesLegend.scss:3]: good catch!

**Pull request merged**